requirements.txt

```diff
@@ -1,10 +1,26 @@
-APScheduler
-boto3
-elasticsearch
-Flask-RESTful
-flask-swagger
-marshmallow
-PyYAML
-setuptools-scm
-uwsgidecorators
-uwsgitop
+aniso8601==9.0.1
+APScheduler==3.7.0
+boto3==1.18.21
+botocore==1.21.21
+certifi==2021.5.30
+click==8.0.1
+elasticsearch==7.14.0
+Flask==2.0.1
+Flask-RESTful==0.3.9
+flask-swagger==0.2.14
+itsdangerous==2.0.1
+Jinja2==3.0.1
+jmespath==0.10.0
+MarkupSafe==2.0.1
+marshmallow==2.21.0
```

Comment:
these are all updated to the latest versions as of the date of this PR except `marshmallow` which had some serious overhauls in v3.0 and would have required rewriting a lot of error-handling